static/js/components/dashboard/CourseAction.js

```diff
@@ -93,7 +93,7 @@ export default class CourseAction extends React.Component {
 
     return (
       <SpinnerButton
-        className="dashboard-button"
+        className="dashboard-button pay-button"
```

Comment:
I think here I just wanted to make the selector in the test more specific. There's no functional change going on here